cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -22,13 +22,17 @@ import (
 
 const (
 	kubeDNSv170AndAboveVersion = "1.14.5"
+	kubeDNSv190AndAboveVersion = "1.14.6"
 )
 
 // GetKubeDNSVersion returns the right kube-dns version for a specific k8s version
 func GetKubeDNSVersion(kubeVersion *version.Version) string {
 	// v1.7.0+ uses 1.14.5, just return that here
```

Comment:
Could update this to reflect current state; for v1.8.x: return 1.14.5, for v1.9.x return 1.14.6